Themes/default/css/index.css

```diff
@@ -57,6 +58,10 @@ textarea, textarea.editor {
 input, button, select {
 	font-size: 0.9em;
 }
+/* Height fixation */
+input, select {
```

Comment:
Yea seems they are lacking class. That side of SMF also needs improvements then. I'll take care after this mid-term exam.